simapp/upgrades/v051/upgrades.go

```diff
@@ -0,0 +1,42 @@
+package v051
+
+import (
+	"context"
+
+	"cosmossdk.io/simapp/upgrades"
+	"cosmossdk.io/x/accounts"
+	protocolpooltypes "cosmossdk.io/x/protocolpool/types"
+	crisistypes "github.com/cosmos/cosmos-sdk/x/crisis/types"
+
+	storetypes "cosmossdk.io/store/types"
+	upgradetypes "cosmossdk.io/x/upgrade/types"
+
+	"github.com/cosmos/cosmos-sdk/types/module"
+)
+
+// UpgradeName defines the on-chain upgrade name
+const UpgradeName = "v0.51"
+
+var Upgrade = upgrades.Upgrade[upgrades.AppKeepers]{
+	UpgradeName:          UpgradeName,
+	CreateUpgradeHandler: CreateUpgradeHandler,
+	StoreUpgrades: storetypes.StoreUpgrades{
+		Added: []string{
+			accounts.StoreKey,
```

Comment:
This fixes a 🐛 as the store key for accounts is not the module name. To be consistent, I have used the store keys for the other types as well